src/lib/network_pool/transaction_pool.ml

```diff
@@ -1387,7 +1387,18 @@ struct
                   in
                   Or_error.errorf "Diff failed with verification failure(s): %s"
                     errs_string
-              | Error _ | Ok () ->
+              | Error errs ->
```

Comment:
revert this change?

Reply:
This change has been reverted in the previous PR.